### PR TITLE
Seller hookup

### PR DESCRIPTION
**Pull request by hpabst**
Finished the work hooking up the seller table cells to the listing view. Changed some UI elements so the search and seller cells match.

### CODE REVIEW

File: TextCrunch/SellerListingViewController.swift [hunk 1]
```diff
@@ -8,23 +8,31 @@
 
 import UIKit
 
-class SellerListingViewController: UIViewController {
+class SellerListingViewController: UIViewController, UITableViewDelegate{
 
+    @IBOutlet weak var inactiveSwitch: UISwitch!
```
Comment (gcoomber) [inline TextCrunch/SellerListingViewController.swift]: All of these weak variables could be listed without spaces between them for readability sake.

File: TextCrunch/SellerListingViewController.swift [hunk 2]
```diff
@@ -39,13 +48,39 @@ class SellerListingViewController: UIViewController {
...
     }
+    
+    func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
+        selectedListing = tableDataSource.currentListings[indexPath.row]
+        self.performSegueWithIdentifier("SellerViewCell", sender: nil)
+    }
+    
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject!) {
+        if (segue.identifier == "SellerViewCell") {
+            var svc = segue.destinationViewController as ListingViewController;
```
Comment (gcoomber) [inline TextCrunch/SellerListingViewController.swift]: Is anything actually being hidden here? Or is this just a segue?

File: TextCrunch/SellerListingViewTableDataSource.swift [hunk 3]
```diff
@@ -53,6 +99,18 @@ class SellerListingViewTableDataSource: NSObject, UITableViewDataSource{
         return cell
     }
     
+    func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
+         var selectedListing = currentListings[indexPath.row] as Listing
+        //This is very ugly, as we're setting this variable then accessing it when performing
+        //a segue. But having a common issue where an exception is thrown where the receiver cannot
+        //identify a segue, even though the segue with the given identifier exists.
+        var targetVC = ListingViewController()
```
Comment (gcoomber) [inline TextCrunch/SellerListingViewTableDataSource.swift]: Could you throw these three lines in the prepare for segue function?

Comment (hpabst) [inline TextCrunch/SellerListingViewTableDataSource.swift]: This code should actually be removed and is leftover from when I was trying to resolve my segue issues.

File: TextCrunch/SellerListingViewController.swift (hunk 2 above)
Comment (hpabst) [inline TextCrunch/SellerListingViewController.swift]: Just a segue, comment is left over from copy-pasting some code and should be removed.

**Pull request merged**